index.html

```diff
@@ -12,38 +12,29 @@
   <div class="landing-page">
     <nav class="nav">
       <div class="nav__logo">my logo</div>
-      <div class="nav__menu"></div>
+      <div class="nav__menu-box">
+        <div class="nav__menu nav__menu--line-1"></div>
+        <div class="nav__menu nav__menu--line-2"></div>
+        <div class="nav__menu nav__menu--line-3"></div>
+      </div>
     </nav>
     <section class="landing">
-      <header class="landing__header">I'm Mariane
-      Glad to see you!</header>
-      <p class="landing__text">
-        I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
-      </p>
-      <div><a href="#" class="landing__link">
-       let's connect
-      </a></div>
-      <div class="land__social-media">
-        <object data="/img/twitter.svg" height="19.99" width="19.99"></object>
-        <object data="/img/Linkedin_icon.svg" height="19.99" width="19.99"></object>
-        <object data="/img/medium.svg" height="19.99" width="19.99"></object>
-        <object data="/img/github.svg" height="19.99" width="19.99"></object>
-        <object data="/img/hands.svg" height="19.99" width="19.99"></object>
-        <!-- <svg class="landing__icon">
-          <use xlink:href="/img/twitter.svg"></use>
-        </svg>
-        <svg class="landing__icon">
-          <use xlink:href="/img/Linkedin_icon.svg"></use>
-        </svg>
-        <svg class="landing__icon">
-          <use xlink:href="/img/medium.svg"></use>
-        </svg>
-        <svg class="landing__icon">
-          <use xlink:href="/img/github.svg"></use>
-        </svg>
-        <svg class="landing__icon">
-          <use xlink:href="/img/hands.svg"></use>
-        </svg> -->
+      <div class="landing__wrapper">
+        <h2 class="landing__header">I'm Mariane
+          Glad to see you!</h2>
+          <p class="landing__text">
+            I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+          </p>
+          <div><a href="#" class="landing__link">
+           let's connect
+          </a></div>
+          <div class="landing__social-media">
+            <object  class="landing__icon" data="/img/twitter.svg" height="20" width="20"></object>
+            <object class="landing__icon" data="/img/Linkedin_icon.svg" height="20" width="20"></object>
+            <object class="landing__icon" data="/img/medium.svg" height="20" width="20"></object>
+            <object class="landing__icon" data="/img/github.svg" height="20" width="20"></object>
+            <object class="landing__icon" data="/img/hands.svg" height="20" width="20"></object>
+      </div>
```

Comment:
- [ ] Please enclose all the icons in a `ul` tag and each icon should have a `li`  tag as well. It is better to enclose groups of related items in a `ul` tag and it is also a requirement for this project 😄 .
- [ ] Each of these icons should also be wrapped in a `a` tag since they are links to your various sites 🏄 .

Reply:
@Lucas-Masaba Thanks for the detailed review. The above updates have been implemented. 

- I found that, When I enclosed the svg icon in an anchor take, the cursor appears about 5px away from the the icon itself.